frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -87,6 +87,20 @@ export const ReportStreamHeader = () => {
                     <span>Submissions</span>
                 </NavLink>
             );
+            // Validate NavLink
+            if (CheckFeatureFlag(FeatureFlagName.VALIDATION_SERVICE)) {
```

Comment:
We cannot put this under admin only unless we change the admin org settings function to set the default sender.
Are we ok with doing that?

Reply:
i have an example of what i mean up... LMK if it is ok!

Reply:
I think for admins it's safe. Right now we aren't mocking senders anywhere, so they'll just use the old method of joining the group in Okta to emulate specific senders. Adding `"default"` will just help us avoid issues when developing with users in mind, since we _can_ reasonably expect them to have a sender name as senders.

Reply:
this was added to admins only and now admins also set a default sender in the org table